custom_components/solarfocus/translations/en.json

```diff
@@ -61,12 +61,12 @@
   "entity": {
     "sensor": {
       "fm_state" : {
-        "state" : {
-          "0": "Vorlauffühler nicht angeschlossen",
-          "1": "Pumpe ausgeschaltet",
-          "2": "Pumpe eingeschaltet",
-          "3": "Handbetrieb aktiv",
-          "4": "Handbetrieb deaktiviert"
+        "state": {
+          "0": "preliminary sensor not connected",
```

Comment:
Should we translate "Vorlauffühler" rather with "Supply sensor"? Similar to "Vorlauftemperatur" -> "Supply temperature"? WDYT?

Reply:
I vote for  "lead water temperature sensor" not connected or "lead temperature sensor"